build_collection.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
 
+VERSION_RELEASE=` echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*'`
```

Comment:
I'd prefer to write this like follows:

`VERSION_RELEASE=$(echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*')`

Reply:
I don't know if you need the extra space:
```
VERSION_RELEASE=$( echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*')
```

Reply:
changed!